packages/pytest-simcore/src/pytest_simcore/simcore_services.py

```diff
@@ -39,6 +39,7 @@
     "static-webserver",
     "traefik",
     "whoami",
+    "notif-worker",

```

Comment:
from the number of characters below, can't we have something more descriptive?
`notifications-wrk`
I would prefer to have the name of the service long and `wrk`/`mng` short